provider-infinispan/src/main/java/io/github/iprodigy/cache/provider/infinispan/InfinispanProvider.java

```diff
@@ -64,6 +66,7 @@ public <K, V> Cache<K, V> build(ICacheSpec<K, V> spec) {
 	}
 
 	@Value
+	@Listener
 	private static class InfinispanListener<K, V> {
```

Comment:
doesn't need to be done in this PR, but would be nice to make these inner classes in providers like this one into full classes